source/templates/partials/_modal-test.twig

```diff
@@ -15,14 +15,30 @@
 
 <div class="form__wrapper">
   <label class="form__label" for=""></label>
-  <input class="form__field" id="" type="date" name="" value="">
+  <input class="form__field" id="" type="text" name="" value="" aria-describedby="calendar">
+  <p id="calendar"></p>
 </div>
 
 <div class="form__wrapper">
-  <select class="form__field" class="field__" id="" name="" size="">
-    <option class="form__field--oprion" value="">Option-1</option>
-    <option class="form__field--oprion" value="">Option-2</option>
+  <label class="form__label" for=""></label>
+  <select class="form__field" id="" name="">
+    <option value="">Option-1</option>
+    <option value="">Option-2</option>
   </select>
 </div>
 
+{# The error state #}
+<div class="form__wrapper">
+  <label class="form__label" for=""></label>
+  <input class="form__field form__fielmd--error" id="" type="text" name="" value="" aria-required="true" aria-describedby="error">
```

Comment:
If the input is in error, we want to be able to style both input AND label. If you just put a class in the input, you'll also need to add a modifier class on the `label`. To simplify it, remove the error class on the input, and add a modifier class on the wrapper. We want to be able to do

```
.form__wrapper--error {
   .form__label {
       // Label styles in error mode
   }
   .form__field {
       // Input styles in error mode
   }
}
```